getting_started/basic_walkthrough.adoc

```diff
@@ -103,27 +107,26 @@ MongoDB pod then starts up and displays its newly-assigned IP address.
 
 In this section, you will configure a GitHub webhook to automatically trigger a
 rebuild of your application whenever you push code changes to your forked
-repository.
+repository. This involves adding the Github webhook URL from your application
+into your Github repository. You obtain this webhook from these locations:
```

Comment:
I'm not sure if the "You obatin this webhook from these locations:" bit is necessary. I know it's moving the bits below up to here, but in doing so I think it makes it unnecessary.